### ISSUE TITLE
swipe up/down the calendar to display weekly/monthly mode

### ISSUE THREAD

**Issue by iad24** [prolificinteractive/material-calendarview#675]
possible to this swiping feature?

![calendar](https://user-images.githubusercontent.com/15226261/31622739-d3986770-b2cf-11e7-8b39-0f3272056598.gif)

**Comment by soujanyamandala** [comment]
Yes. Currently the library supports only horizontal swipe not the vertical.

**Comment by iad24** [comment]
the horizontal swipe is for the pagination of the weeks right?
The one above is to switch from monthly to weekly mode and vice versa

**Comment by soujanyamandala** [comment]
Yes. I am also looking for something that switches monthly to weekly mode on swipe up and down.

**Comment by sweety000** [comment]
YEs, me too waiting for this feature. If you added this please let me know.
